pkg/client/informers/externalversions/cr/v1alpha1/repositoryserver.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2023 The Kanister Authors.
+Copyright The Kubernetes Authors.
```

Comment:
Same as above - we should change it back

Reply:
fixed